src/identity/tezos/TezosLedgerConnector.ts

```diff
@@ -0,0 +1,110 @@
+import Transport from "@ledgerhq/hw-transport";
+import * as bip32path from 'bip32-path';
+
+export enum Curve { // taken from https://github.com/obsidiansystems/ledger-app-tezos/blob/master/APDUs.md
+    ED25519 = 0x00,
+    SECP256K1 = 0x01,
+    SECP256R1 = 0x02
+};
+
+export enum Instruction { // taken from https://github.com/obsidiansystems/ledger-app-tezos/blob/master/APDUs.md
+    INS_VERSION = 0x00,
+    INS_GET_PUBLIC_KEY = 0x02,
+    INS_PROMPT_PUBLIC_KEY = 0x03,
+    INS_SIGN = 0x04,
+    INS_SIGN_UNSAFE = 0x05
+}
+
+export default class TezosLedgerConnector {
+    transport: Transport<*>;
+
+    constructor(transport: Transport<*>) {
+        this.transport = transport;
+        transport.decorateAppAPIMethods(this, ["getAddress", "signOperation", "signHash", "getVersion"], "XTZ");
+    }
+
+    /**
+     * Get Tezos public key hash for a given BIP32/44 path and curve
+     * 
+     * @param {string} path BIP32/44 derivation path
+     * @param {boolean} prompt Prompt the user to provide the key on the hardware device, default true.
+     * @param {Curve} curve Curve to use for key generation, one of: ED25519 (default), SECP256K1, SECP256R1
+     */
+    async getAddress(path: string, prompt: boolean = true, curve: Curve = Curve.ED25519): Promise<string> {
+        if (!path.startsWith("44'/1729'")) {
```

Comment:
Is there a reason for stopping people completely from using their own custom schemes?